client/galaxy/scripts/components/Panels/ToolBox.vue

```diff
@@ -106,6 +115,7 @@ export default {
     methods: {
         onQuery(query) {
             this.query = query;
+            this.queryFilter = (query && (query.length >= 3)) ? query : null;
```

Comment:
Do you need this? The `ToolSearch` component has `minQueryLength` already set to `3`.